### PR TITLE
adapt node changes for evmreader

### PR DESCRIPTION
**Pull request by ZzzzHui**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0]
```
Comment (miltonjonat) [inline CHANGELOG.md:9]:
```suggestion
## [Unreleased]
```
The CHANGELOG does have a link at the end for `[UNRELEASED]`, that currently points to [rollups-espresso-reader/compare/v0.2.2-node-20250128...HEAD](https://github.com/cartesi/rollups-espresso-reader/compare/v0.2.2-node-20250128...HEAD)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+
+- Adapted to the node's way of fixing epoch closure bug
```
Comment (miltonjonat) [inline CHANGELOG.md:13]:
```suggestion
- Improved epoch closure procedure that follows Node v2's strategy and avoids edge case that can lead to missing an L1 block
```

**Review by miltonjonat — COMMENTED** [review]
Changes seem to make sense, doing some tests here. In the meantime, I suggested some things for the CHANGELOG updates ;)
